src/test/java/sirius/kernel/nls/NLSSpec.groovy

```diff
@@ -106,68 +106,29 @@ class NLSSpec extends BaseSpecification {
         result == ""
     }
 
-    def "toSpokenDate() formats today as today"() {
-        given:
-        def date = LocalDate.now()
-        when:
-        def result = NLS.toSpokenDate(date)
-        then:
-        result == "heute"
-    }
-
-    def "toSpokenDate() formats yesterday as yesterday"() {
-        given:
-        def date = LocalDate.now().minusDays(1)
-        def dateTime = LocalDateTime.now().minusDays(1)
-        when:
-        def result = NLS.toSpokenDate(date)
-        def resultTime = NLS.toSpokenDate(dateTime)
-        then:
-        result == "gestern"
-        resultTime == "gestern"
-    }
-
-    def "toSpokenDate() formats tomorrow as tomorrow"() {
-        given:
-        def date = LocalDate.now().plusDays(1)
-        def dateTime = LocalDate.now().plusDays(1)
-        when:
-        def result = NLS.toSpokenDate(date)
-        def resultTime = NLS.toSpokenDate(dateTime)
-        then:
-        result == "morgen"
-        resultTime == "morgen"
-    }
-
-    def "toSpokenDate() formats a date in the past as date"() {
-        given:
-        def date = LocalDate.of(2014, 1, 1)
-        when:
-        def result = NLS.toSpokenDate(date)
-        then:
-        result == "01.01.2014"
-    }
+    def "toSpokenDate() formats dates and dateTimes correctly"() {
+        expect:
+        NLS.toSpokenDate(date) == output
 
-    def "toSpokenDate() formats a date in the future as date"() {
-        given:
-        def date = LocalDate.of(2114, 1, 1)
-        when:
-        def result = NLS.toSpokenDate(date)
-        then:
-        result == "01.01.2114"
-    }
+        where:
+        date                                 | output
+        LocalDate.now()                      | "heute"
+        LocalDateTime.now()                  | "vor wenigen Minuten"
+        LocalDate.now().minusDays(1)         | "gestern"
+        LocalDateTime.now().minusDays(1)     | "gestern"
+        LocalDate.now().plusDays(1)          | "morgen"
+        LocalDateTime.now().plusDays(1)      | "morgen"
+        LocalDate.of(2114, 1, 1)             | "01.01.2114"
+        LocalDateTime.of(2114, 1, 1, 0, 0)   | "01.01.2114"
+        LocalDate.of(2014, 1, 1)             | "01.01.2014"
+        LocalDateTime.of(2014, 1, 1, 0, 0)   | "01.01.2014"
+        LocalDateTime.now().minusMinutes(5)  | "vor wenigen Minuten"
+        LocalDateTime.now().minusMinutes(35) | "vor 35 Minuten"
+        LocalDateTime.now().minusHours(1)    | "vor einer Stunde"
+        LocalDateTime.now().minusHours(4)    | "vor 4 Stunden"
+        LocalDateTime.now().plusMinutes(40)  | "in der nächsten Stunde"
+        LocalDateTime.now().plusHours(4)     | "in 3 Stunden"
```

Comment:
ich versteh zwar wieso, aber des ist trotzdem bisle verwirrend. 
.minusHours(4) -> vor 4 Stunden
.plusHours(4) -> in 3 Stunden

Reply:
weil halt eine Nanosekunde oder so vergeht wird aus dem zweiten "in 3 Stunden" - ich hatte mir überlegt anstatt 4 hours; 4 hours und 1minute draufzurechnen - das fand ich dann aber auch verwirrend ?

Reply:
man könnte ja generell normal runden statt abzurunden oder?

Reply:
das würde den Code unnötig komplizieren da auf die Minuten zu gucken. Wirklich betrifft das wirklich ja nur diesen einen Test und der ist ja nur beim ersten mal lesen verwirrend.

Reply:
kannst ja ein comment hinter den testcase machen